package.json

```
@@ -17,8 +17,8 @@
     "standard"
   ],
   "engines": {
-    "node": ">=16.0.0",
-    "npm": ">=7.0.0"
+    "node": ">=18",
+    "npm": ">=9"
```

See other PR comment for discussion.

```suggestion
    "node": ">=18.14.0",
    "npm": ">=9.3.1"
```